app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -87,6 +89,12 @@ public ListAdapter getAdapter() {
 
     public void setAdapter(ListAdapter adapter) {
         this.contributionsList.setAdapter(adapter);
+        Timber.d("" + adapter.getCount());
```

Comment:
It seems these debug logs were added during development. These can be removed now. :)

Reply:
Yup, will do it :+1: